core-bundle/src/Security/ContaoCorePermissions.php

```diff
@@ -186,4 +186,39 @@ final class ContaoCorePermissions
      * Subject must be a corresponding group ID or an array of group IDs.
      */
     public const MEMBER_IN_GROUPS = 'contao_member.groups';
+
+    /**
+     * Prefix for all DCA related permission attributes.
+     */
+    public const DCA_PREFIX = 'contao.dca.';
```

Comment:
I find that prefix constant a little superfluous if only used by the voter and tests. We didn't add one for `contao_member.` either. But I can live with it if you insist 🙃

Reply:
I would prefer having it because I want to use it also when looping over `operations`, `global_operations` and `buttons` which can have an arbitrary key but they should all run in this "namespace".